databricks_cli/unity_catalog/delta_sharing_cli.py

```diff
@@ -136,6 +136,9 @@ def shared_data_object(name):
 @click.option('--add-table', default=None, multiple=True,
               metavar='NAME',
               help='Full name of table to add to share (can be specified multiple times).')
+@click.option('--update-table', default=None, multiple=True,
```

Comment:
You're not updating anything here, though...

Note that the other add/remove table flags are basic flags without CDF, partitioning, and aliasing options.

Reply:
Ah oops. I assumed that it would create the object automatically!